tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "module": "esnext",
     "lib": ["dom", "esnext"],
```

Comment:
I believe we should add `dom.iterable` here, otherwise we won't get annotations for all of [these](https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.iterable.d.ts) types

Also be aware that create-react-app [base tsconfig](https://github.com/tsconfig/bases/blob/main/bases/create-react-app.json) adds it

Reply:
added